app/controllers/application_controller.rb

```diff
@@ -1,20 +1,32 @@
 class ApplicationController < ActionController::Base
+    include CanCan::ControllerAdditions
+    rescue_from CanCan::AccessDenied, with: :unauthorized_access
+
+
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
-  before_action :authorize_admin
+ # before_action :authorize_admin
   before_filter :set_vars
   protect_from_forgery with: :exception
+  before_action :authenticate_user!
 
 
   protected
-  	def authorize_admin
-  		unless User.find_by(id: session[:user_id])
-  			redirect_to login_url, notice: "Please log in"
-  		end
-  	end
+  #	def authorize_admin
+  #		unless User.find_by(id: session[:user_id])
+  #			redirect_to login_url, notice: "Please log in"
+  #		end
+  #	end
 
   	def set_vars
```

Comment:
remove empty method

Always take care of code indentation, your code should look beautiful